lib/main_staging.dart

```diff
@@ -21,6 +22,7 @@ void main() async {
     await getIt<FirebaseService>().initialize();
   }
   await getIt<MatrixService>().init();
+  await getIt<AIClient>().init();
```

Comment:
don't need to call init here, let put logic of init function into AIClient constructor call